website/client/components/guilds/groupFormModal.vue

```diff
@@ -0,0 +1,324 @@
+
+  .item-with-icon {
+    display: inline-block;
+
+    img {
+      height: 20px;
+      margin-right: .5em;
+    }
+
+    .count {
+      font-size: 14px;
+      font-weight: bold;
+      margin-right: 1em;
+      color: $green-10;
+    }
+  }
+
+  .description-count {
+    margin-top: 1em;
+  }
+
+  .category-select {
+    border-radius: 2px;
+    background-color: #ffffff;
+    box-shadow: 0 2px 2px 0 rgba(26, 24, 29, 0.16), 0 1px 4px 0 rgba(26, 24, 29, 0.12);
+    padding: 1em;
+  }
+
+  .category-select:hover {
+    cursor: pointer;
+  }
+
+  .category-wrap {
+    margin-top: .5em;
+  }
+
+  .icon {
+    margin-left: .5em;
+    display: inline-block;
+  }
+</style>
+
+<script>
+import bModal from 'bootstrap-vue/lib/components/modal';
+import bBtn from 'bootstrap-vue/lib/components/button';
+import bFormInput from 'bootstrap-vue/lib/components/form-input';
+import bFormCheckbox from 'bootstrap-vue/lib/components/form-checkbox';
+import bFormSelect from 'bootstrap-vue/lib/components/form-select';
+import bTooltip from 'bootstrap-vue/lib/components/tooltip';
+
+export default {
+  components: {
+    bModal,
+    bBtn,
+    bFormInput,
+    bFormCheckbox,
+    bFormSelect,
+    bTooltip,
+  },
+  data () {
+    let data = {
+      newGuild: {
+        id: '',
+        name: '',
+        type: 'guild',
+        privacy: 'private',
+        description: '',
+        guildInformation: '',
+        categories: [],
+        onlyLeaderCreatesChallenges: true,
+        guildLeaderCantBeMessaged: true,
+        privateGuild: true,
+        allowGuildInvationsFromNonMembers: true,
+        newLeader: '',
+      },
+      categoryOptions: [
+        {
+          label: 'animals',
+          key: 'animals',
+        },
+        {
+          label: 'artDesign',
+          key: 'art_design',
+        },
+        {
+          label: 'booksWriting',
+          key: 'books_writing',
+        },
+        {
+          label: 'comicsHobbies',
+          key: 'comics_hobbies',
+        },
+        {
+          label: 'diyCrafts',
+          key: 'diy_crafts',
+        },
+        {
+          label: 'education',
+          key: 'education',
+        },
+        {
+          label: 'foodCooking',
+          key: 'food_cooking',
+        },
+        {
+          label: 'healthFitness',
+          key: 'health_fitness',
+        },
+        {
+          label: 'music',
+          key: 'music',
+        },
+        {
+          label: 'relationship',
+          key: 'relationship',
+        },
+        {
+          label: 'scienceTech',
+          key: 'science_tech ',
+        },
+      ],
+      showCategorySelect: false,
+      members: ['one', 'two'],
+    };
+
+    let hashedCategories = {};
+    data.categoryOptions.forEach((category) => {
+      hashedCategories[category.key] = category.label;
+    });
+    data.categoriesHashByKey = hashedCategories;
+
+    return data;
+  },
+  mounted () {
+    let vm = this;
+    this.$root.$on('shown::modal', () => {
```

Comment:
when you use `=>` to define a function the value of `this` inside the function is the same as outside of it so `let vm = this` is not necessary in this case